monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Para máxima seguridad — seguridad incluso si tu dispositivo primario está comprometido por malware — se pueden utilizar monederos hardware especializados. Los monederos hardware son pequeños dispositivos electrónicos que guardan tus claves privadas y se comunican con software en el dispositivo primario usando sólo información pública. Los monederos hardware otorgan seguridad extra por el típico precio de $50-$150.
+
+Así que, con todo esto en mente, considera las siguientes opciones: si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual. Ambos pueden ser configurados para ya sea ejecutar un nodo completo o conectarse a un nodo remoto en un modo mejorado de privacidad. Estos pueden ser descargados de [getmonero.org/es/downloads](https://getmonero.org/es/downloads/). Para dispositivos Android, Monerujo, en [monerujo.io](https://www.monerujo.io/), o para iOS, Cake Wallet, en [cakewallet.io](https://cakewallet.io/), son monederos ligeros de protección de claves de visualización. Si quisieras un monedero ligero de compartir claves de visualización, el monedero basado en navegador MyMonero, en [mymonero.com](https://mymonero.com/), se recomienda. The Edge Wallet, en [edge.app](https://edge.app/), es otra sólida opción de monedero ligero de compartir claves de visualización para ambos Android y iOS.
+
+Al escoger entre estos, ten en cuenta que la mayoría tiene una opción extra en la cual — siendo la excepción The Edge Wallet — pueden ser configurados para conectarse ya sea a un servidor remoto de terceros o a un servidor que ejecutes con un nodo. Esto se ilustra en la tabla.
+
+## Monederos Oficiales
+
++ [getmonero.org/es/downloads](https://www.getmonero.org/es/downloads)  
+Si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual.
```

Comment:
Suggestion:

"El primero provee una interfaz gráfica de usuario, y el _**segundo**_ una interfaz textual."

Reply:
It's the same but I'll take the suggestion.

Reply:
Anyway you prefer :)